js task 1/home.js

```diff
@@ -0,0 +1,14 @@
+const str ='develop';
+function slicee(str){
+    const slicee = str.slice(-3);
+    console.log(slicee);
+    return slicee;
+
+
+
+}
+if(str.length >= 3){
+    console.log(slicee(str)+str+slicee(str));
```

Comment:
You are calling slicee(str) twice. It would be more efficient to call it once and store the result in a variable to avoid redundant operations

Reply:
Although you handle the case where the string length is greater than or equal to 3, there is no fallback in case the string length is less than 3. Consider adding a fallback to improve robustness